src/main/java/net/minestom/server/entity/LivingEntity.java

```diff
@@ -607,6 +607,13 @@ public void refreshActiveHand(boolean isHandActive, boolean offHand, boolean rip
         }
     }
 
+    public void leaveBed() {
```

Comment:
perhaps should add `enterBed` or `sleep` taking a position to mimic this method.

Reply:
done